react-client/yarn-error.log

```diff
@@ -0,0 +1,6836 @@
+Arguments: 
```

Comment:
does this need to be commited?

Reply:
ahhh good point. i'll update the gitignore.